content/automate/ManageIQ/Transformation/TransformationHosts/Common.class/__methods__/utils.rb

```diff
@@ -79,24 +79,23 @@ def self.remote_command(task, transformation_host, command, stdin = nil, run_as
 
             def self.virtv2vwrapper_options_vmwarews2rhevm_vddk(task)
               source_vm = task.source
-              source_ems = source_vm.ext_management_system
               source_cluster = source_vm.ems_cluster
 
               destination_cluster = task.transformation_destination(source_cluster)
               destination_ems = destination_cluster.ext_management_system
               destination_storage = task.transformation_destination(source_vm.hardware.disks.select { |d| d.device_type == 'disk' }.first.storage)
 
-              vmware_uri = "vpx://"
-              vmware_uri += "#{source_ems.authentication_userid.gsub('@', '%40')}@#{source_ems.hostname}/"
+              vmware_uri = "esx://"
+              vmware_uri += "root@#{source_vm.host.ipaddress}/"
```

Comment:
Is it ok to always use `root`?  I believe that can be disabled on ESX.  Will this be noted as a requirement?

Reply:
root user is always enabled on the ESXi host when using the API. It's not the SSH connection.